docs/user-guide/reading-and-writing-files.ipynb

```diff
@@ -0,0 +1,83 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reading and Writing Files\n",
+    "\n",
+    "## HDF5\n",
+    "\n",
+    "Scipp support writing variables, data arrays, and dataset to HDF5 files.\n",
+    "Reading of HDF5 is supported *only* for these scipp-specific files.\n",
+    "Other HDF5-based formats are not supported at this point.\n",
+    "For reading NeXus files, see [scipp-neutron](../scipp-neutron/overview.html).\n",
```

Comment:
https://www.nexusformat.org/ especially given this is in the generic (non-neutron-scattering docs for scipp)